src/components/mypage/ReviewModal/index.tsx

```diff
@@ -0,0 +1,135 @@
+import type { ChangeEventHandler, ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type {
+  ReviewModalProps,
+  EvaluateState,
+  EvaluateData,
+  ReviewState
+} from './types'
+
+const evaluateData: EvaluateData = [
```

Comment:
`evaluate`가 서비스에서 생소한 단어인 것 같아서 `score` 정도로 표현해도 괜찮을 것 같은데 어떠신가요?
+ BE에서는 어떻게 사용되는지 여쭤봐도 될까요??

```suggestion
const MOCK_SCORE: Score = [
```

상수 mock은 대문자로 표현되면 좋을 것 같습니다

Reply:
넵 score가 더 일반적이고 좋은 것 같습니다 ㅎㅎ
<img width="890" alt="image" src="https://github.com/price-offer/offer-fe/assets/66211721/1ee089df-7836-4689-8151-185ca7d7816c">
백엔드도 스코어네요!!